<launch>
    <node name="spcu" pkg="spcu" type="spcu">
        <param name="port_name" value="/dev/ttyUSB0" />
        <param name="publish_rate" value="0.1" />
    </node>
</launch>
